source/managing_account/02_generating_api_key/generating_api_key.rst

```diff
@@ -0,0 +1,30 @@
+Generating an API key
+=====================
+
+From the API keys tab of their account, authenticated users have the possibility to generate API keys, as well as to manage them.
+
+These API keys allow to use the following consultation APIs: ODS Search API v1 and v2, OData, WFS and CSW (more info in the :doc:`Using APIs </exploring_catalog_and_datasets/04_getting_involved/using_api>` documentation).
```

Comment:
Is `consultation APIs` something we use in other places? That sounds a bit awkward to me but why not if it's coherent with other parts of the documentation